src/autorun.ts

```diff
@@ -177,40 +187,44 @@ export const computed = <T>(fn: Cb<T>): Observable<T> => new Observable(observer
                 if (v.hasValue) {
                     return v.value as O;
                 } else {
-                    throw ERROR_STUB;
+                    throw HALT_ERROR;
                 }
             }
 
             const v: TrackEntry<O> = {
                 hasValue: false,
                 value: void 0,
-                subscription: void 0,
+                // eagerly create subscription that can be destroyed
+                subscription: new Subscription(),
```

Comment:
This was rather a precaution than a need:
if expression unsubscription would happen synchronously _(I don't think it can happen now)_ - this TrackEntry won't have a subscription to be destroyed.

Reply:
Can't look right now cause no laptop on my vacation, but if I remember correctly it uses the coalescing operator (?.) to unsubscribe this right? So I'm not sure this is needed...

Reply:
You're right! It was a precaution against something that doesn't exist: if we unsubscribe from all `deps` synchronously in subscription — then this subscription is not there yet. It's a tricky scenario but it seems to be currently impossible 😅

_Also for my "fix" to work, I'd need to also move adding `track entry` to deps before `o.subscribe(…)`_

Have a rest, take your sunbath, stop checking github 😛😄